portfolio/src/main/java/com/google/sps/servlets/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.google.sps.servlets;
+
+public final class Constants {
+  private Constants() {}
+  public static final String stats = "/WEB-INF/cases.csv";
+  public static final String cType = "application/json";
+  public static final String encoding = "UTF-8";
+  public static final String says = "says ";
```

Comment:
I feel not all these constants need to be defined in a separate file, unless they need to be used in more than one file. For example, "cType" and "encoding" might be used somewhere else. You can move other constants to the class where they are used.